README.md

```diff
@@ -53,38 +53,47 @@ The tab strip which will contain the tab items
   android:layout_width="match_parent" />
 ```
 
-Attaching the `ViewPager` to the `LandingStrip` and handling tab clicks.
+A `TabAdapter` implementation to set the tab titles from `PagerAdapter.getPageTitle`
 
 ```java
-final ViewPager viewPager = (ViewPager) findViewById(R.id.view_pager);
-final LandingStrip landingStrip = (LandingStrip) findViewById(R.id.landing_strip);
-viewPager.addOnPageChangeListener(landingStrip);
-landingStrip.setAdapter(new LandingStripAdapter<TextView>() {
+static class TitledAdapter extends TabAdapter<TextView> {
 
-    private final String titles[] = {"hello", "world"};
+    private final ViewPager viewPager;
 
-    @Override
-    protected TextView createView(ViewGroup parent, int position) {
-        return (TextView) getLayoutInflater().inflate(R.layout.tab_simple_text, parent, false);
+    public TitledAdapter(LayoutInflater layoutInflater, int itemLayoutId, LandingStrip landingStrip, ViewPager viewPager) {
+        super(layoutInflater, itemLayoutId, landingStrip, viewPager);
+        this.viewPager = viewPager;
     }
 
     @Override
-    protected void bindView(TextView view, final int position) {
-        view.setText(titles[position]);
-        view.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                landingStrip.setCurrentItem(position);
-                viewPager.setCurrentItem(position);
-            }
-        });
+    protected void bindView(TextView view, int position) {
+        super.bindView(view, position);
```

Comment:
we don't _neeeed_ to call the `super`, the idea was if somebody wants to change behaviour but doesn't want to implement the full `BaseAdapter` for some reason~ they can. 

Skipping `super` : 
- `createView` will disable the tabLayout from being inflated.
- `bindView` will disable the tab `clickListener` from binding.
- `getCount` will disable the count being read from the `PagerAdapter.getCount`   
   
If this class was the only way to create a `Adapter` I would agree with you for `final`ing and forcing the `supers` to be called but it's just a helper.  If someone forgets to call `super` they've broken the contract of overriding a method.